third-party-programs.txt

```diff
@@ -85,16 +81,14 @@ zlib License
      misrepresented as being the original software.
   3. This notice may not be removed or altered from any source distribution.
 
-  Jean-loup Gailly        Mark Adler
-  jloup@gzip.org          madler@alumni.caltech.edu
 
 --------------------------------------------------------------------------------
 
-3.  Nbody
-
-MIT License
+3. Nbody
+   (c) 2019 Fabio Baruffa
```

Comment:
Did you intend to delete the word 'Copyright' here?

Reply:
No need to review, this was specified specifically from the legal team, merely a copy/paste replacement

Reply:
OK, rubber stamp approval on the way :-)